pages.pt_BR/linux/wtf.md

```diff
@@ -0,0 +1,12 @@
+# wtf
+
+> Mostra a expansão de acrônimos.
+> Mais informações (em inglês): <https://manpages.debian.org/bsdgames/wtf.6.en.html>.
+
+- Expandir um acrônimo:
```

Comment:
You shouldn't use infinite, you should use indicative mood, I think you should use:
```suggestion
- Expanda um acrônimo:
```
I could be wrong with this conjugation :grin: